core/src/main/java/org/eclipse/dash/licenses/MavenIdParser.java

```diff
@@ -50,16 +52,16 @@ public IContentId parseId(String value) {
 		String artifactid = matcher.group("artifactid");
 		String version = matcher.group("version");
 
-		String type = groupid.startsWith("p2.eclipse-") ? "p2" : "maven";
-		String source = groupid.startsWith("p2.eclipse-") ? "orbit" : "mavencentral";
+		String type = groupid.startsWith(TychoConstants.P2_GROUPID_PREFIX) ? "p2" : "maven";
```

Comment:
By the way: From Tycho 2.7.on the output of the type gives a clear indication of the actual type:

p2.eclipse.plugin:org.w3c.dom.smil:**eclipse-plugin**:1.0.1.v200903091627:system

Reply:
"Type" here is in the context of ClearlyDefined IDs. From an intellectual property management point of view, we don't generally care about the distinction between an Eclipse Platform plug-in or feature.